### PR TITLE
Add first outline for V1 release blogpost

### PR DESCRIPTION
**Pull request by flomotlik**
This outline still uses rough language and is only meant to show what the content will be. Please review on content, not on language or writing style specifically. Once we're happy with the content I'll change the text to be nice english.

### CODE REVIEW

File: posts/2016-10-10-serverless-v1.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+# Serverless 1.0 Release
+
+Very happy to announce final release of Serverless 1.0. We worked very hard on making Serverless an amazing tool over the last year and this is the culmination of many conversations with many teams.
+
+Serverless should be best tool to manage FaaS infrastructure for any team. From small to large to enterprise we see so many jump on FaaS and Serverless. They all get the advantage of focusing fully on their business code and not having to do operations for many things in their infrastructure.
+
+To get started all you really need to do is run the following commands (or watch this video):
+
+```
```
Comment (DavidWells) [inline posts/2016-10-10-serverless-v1.md]: Digging the outline =)

Just FYI/Quick note on code snippets. 

Use the language with the code block to get syntax highlighting just like on github. aka `bash` here

"```bash
serverless rocketlaunch -target moon
```
"

**Comment by teddylun** [comment]
🙈 Sorry teddylun, you aren't authorized to schedule posts.

You must be a collaborator on this repository to schedule posts.

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Comment by teddylun** [comment]
🙈 Sorry teddylun, you aren't authorized to schedule posts.

You must be a collaborator on this repository to schedule posts.

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Comment by teddylun** [comment]
🙈 Sorry teddylun, you aren't authorized to schedule posts.

You must be a collaborator on this repository to schedule posts.

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request closed without merging**